index.d.ts

```diff
@@ -7,9 +7,11 @@
  * @param options all info for this parsing operation
  * @param variant the optional \uFE0F ("as image") variant, in case this info is anyhow meaningful. By default this is ignored.
  */
-declare type ParseCallback = (icon: string, options: object, variant: string) => string | false;
```

Comment:
this should probably stay exported, in case someone is splitting out their parse callback to be a function on its own line and they want to use this type for it. similar for `TwemojiOptions` and `ReplacerFunction` below

Reply:
declare keyword is actually optional for type and interface and do nothing. https://github.com/Microsoft/TypeScript/issues/1940#issuecomment-98888362.

Reply:
oh cheers, TIL. slightly different from Flow here then :)